src/docs/quick-start.md

```diff
@@ -2,196 +2,167 @@
 title: Quick Start Tutorial
 ---
 
-Railway is a deployment platform where you can provision infrastructure,
-develop with that infrastructure locally, and then deploy to the cloud.
+Railway is a deployment platform that lets you provision infrastructure, develop locally with that infrastructure, and deploy to the cloud or simply run ready-made software from our template marketplace.
 
-In this guide, we will deploy a simple To-Do App running [NextJS](https://nextjs.org/docs/getting-started) and [Prisma](https://www.prisma.io/docs/) using a template.
+**In this guide we will cover two different topics to get you quickly started with the platform -**
 
-<Image src="/images/getting-started/1-railway-getting-started.gif"
-alt="Preview of What The Guide is Building"
-layout="intrinsic"
-width={800} height={455} quality={100} />
+1. **Deploying your project** - Bring your code and let Railway handle the rest.
 
-We will get you up and running quickly by covering the following three steps:
+    **[Option 1](/quick-start#deploying-your-project---from-github)** - Deploying from GitHub.
 
-1. Creating a project from a template
-2. Developing the project locally
-3. Deploying to the cloud
+    **[Option 2](/quick-start#deploying-your-project---with-the-cli)** - Deploying with the [CLI](/guides/cli).
 
-You will need [Node](https://nodejs.org/en/download/) to run the project locally. And you will need either Node or [Homebrew](https://brew.sh/) on your machine to install the Railway CLI. We will use [Yarn](https://yarnpkg.com/) for our examples locally.
+3. **Deploying a template** - Ideal for deploying pre-configured software with minimal effort.
 
-## Create a Project
+For our project, we'll utilize a basic <a href="https://github.com/railwayapp-templates/nextjs-basic" target="_blank">NextJS app</a>, and for the template deployment, we'll use the <a href="https://railway.app/template/umami-analytics" target="_blank">Umami template</a>.
```

Comment:
This is a bit confusing to me.  I'm wondering what it means by "our project" and what is this NextJS app?  I think simply expanding on what exactly this is walking me through would be good.  like "To accomplish the first option above, this guide will demonstrate the process for deploying a project directly from Github"

I think you should also mention that they'll need to link their Github with Railway maybe?  just imagining, what if I'm a user who is using this as a step-by-step guide and I haven't done that part yet.  What if we even included a step that's like "fork this repository" so they can follow along?